recipes/limix-legacy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "limix-legacy" %}
+{% set version = "0.8.10" %}
+{% set sha256 = "9e9e5a66283272536001d8f6fc90c4b36ae9d9b0cc34a63017549f76050eecac" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy >=1.10
```

Comment:
It looks like this might require `numpy x.x` in both the `build` and `run` sections since it is including numpy in the cython extension modules:

http://conda-forge.github.io/docs/meta.html#building-against-numpy